website/settings.py

```diff
@@ -51,6 +51,13 @@
 SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC = "https://custombuilder.poly.edu/custom_install/xmlrpc/" # Default, currently no repy_v2 support
 #SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC = "https://sensibilityclearinghouse.poly.edu/custominstallerbuilder/xmlrpc/" # SensibilityTestbed's CIB, which supports repy_v2 and thereby NAT traversal.
 
+# In Python 2.7.9+ a HTTPS request to a server using a self-signed certificate
+# will raise an exception.
+# This setting provides to chose whether to accept self-signed certificates
+# (e.g. in DEBUG mode) by using an unverified ssl context.
+# c.f. https://github.com/SeattleTestbed/custominstallerbuilder/issues/16
+UNVERIFIED_SSL_CONTEXT = DEBUG
```

Comment:
I suppose `UNVERIFIED_SSL_CONTEXT` can be set to True independent of DEBUG.  In that case, I can imagine someone doing so and failing to notice that self-signed certificates are used in production (because they forgot it was set to True).

Consider having some type of warning for when self-signed certificates are accepted by the server.

Reply:
I think that that's not necessary, since the variable is currently tied to the debug mode and shouldn't really be set independently anyway.

Reply:
Why have a separate UNVERIFIED_SSL_CONTEXT setting?  Perhaps you should make it so that DEBUG results in the ability to use unverified SSL contexts.

Someone might try to modify if you don't at least explain that it shouldn't be independently modified.

Reply:
I proposed the separate setting mostly for transparency. We never instruct a *ch operator* to set this to a different value than `DEBUG` and I can't see why a *ch operator* would want to do this, or would do this by accident.